recipe/meta.yaml

```diff
@@ -1,34 +1,36 @@
 {% set name = "bw2data" %}
-{% set version = "4.0.dev21" %}
+{% set version = "4.0.dev22" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 41188a5ed32908ed6a442418689918c988ba289deede118d238f0e79cd52dabb
+  sha256: 210cd87819a9d469b3d842b9c8ca72db196ace049693901af6ef77c5b7d5528f
 
 build:
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
   entry_points:
     - bw2-uptodate = bw2data.bin.bw2_uptodate:main
 
 requirements:
   host:
-    - python >=3.4
+    - python >=3.9
     - pip
     - setuptools
   run:
     - bw2parameters
+    - bw2calc >=2.0.dev13
```

Comment:
To make the package use another pre-release verison you have to follow the guide at https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
Thanks! Actually, we will just drop this instead: https://github.com/brightway-lca/brightway2-data/issues/145